### ISSUE TITLE
azure-mgmt-netapp: Operation: 'Change Ownership' is not permitted when creating NFSv4.1 volume

### ISSUE THREAD

**Issue by lonico** [Azure/azure-sdk-for-python#20356]
- **Package Name**: azure-mgmt-netapp
- **Package Version**: 4.0.0
- **Operating System**: linux
- **Python Version**: 3.7

**Describe the bug**
It is not possible to successfully create a NFSv4.1 volume using the new version of the SDK, if the change ownership mode feature is not enabled.  The volome is created but fails to create a mount point and reports
```
Operation: 'Change Ownership' is not permitted.
```

NFSv4.1 requires to create an ExportPolicyRule.
Unfortunately, it forces a default for chown_mode, which triggers the error.

**To Reproduce**
1. set options = dict(rule_index=1, allowed_clients='0.0.0.0/0')
2. create VolumePropertiesExportPolicy(rules=[ExportPolicyRule(**options)])
3. create a volume with following properties

```
'location': 'eastus', 
'creation_token': 'tests-volume2-ln',

'protocol_types': ['NFSv4.1'],
'service_level': 'Standard',
'tags': None,
'usage_threshold': 107374182400,
'export_policy': VolumePropertiesExportPolicy(rules=[ExportPolicyRule(**options)]),
```

Note that the type is reported as
'type': 'Microsoft.NetApp/netAppAccounts/capacityPools/volumes', 

**Expected behavior**
The volume is created with success.

**Screenshots**

**Additional context**
I was able to identify the root of the problem to:
azure/mgmt/netapp/models/_models_py3.py

I don't think we should force a default for chown_mode.   Whether I used Restricted or Unrestricted, I see the issue.

By changing 
```
chown_mode: Optional[Union[str, "ChownMode"]] = "Restricted",
```
to
```
chown_mode: Optional[Union[str, "ChownMode"]] = None,
```

I was able to bypass the issue and create a volume and the mount point.

**Comment by xiangyan99** [comment]
Thanks for the feedback, we’ll investigate asap.

**Comment by RAY-316** [comment]
hi, @lonico azure-mgmt-netapp have published a new version 5.0.0, you can take a look whether it meet your requirments
